.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
 include:
   - project: "f5/nginx/kic/kic-pipelines"
     file: "/include/ingress-controller.yml"
-    ref: "master"
```

Comment:
We can change this back to master before merging

Reply:
done